vscode/src/custom-prompts/utils/menu.ts

```diff
@@ -1,33 +1,22 @@
-import { QuickPickItem } from 'vscode'
+import { commands, QuickInputButton, QuickInputButtons, QuickPickItem, ThemeIcon, window } from 'vscode'
 
 import { CodyPrompt } from '@sourcegraph/cody-shared'
+import { CodyPromptType } from '@sourcegraph/cody-shared/src/chat/prompts'
 
-export const CodyMenu_CodyCommands = {
-    title: 'Cody Commands',
-    placeHolder: 'Search for a command',
-    ignoreFocusOut: true,
-}
-
-export const CodyMenu_CodyCustomCommands = {
-    title: 'Cody Custom Commands (Experimental)',
-    placeHolder: 'Search command to run...',
-    ignoreFocusOut: true,
-}
-
-export const CodyMenu_NewCustomCommands = 'Cody Custom Commands (Experimental) - New User Command'
+import { ContextOption } from '.'
 
-export const CodyMenu_CodyCustomCommandsConfig = {
-    title: 'Configure Custom Commands (Experimental)',
-    placeHolder: 'Choose an option',
+export const NewCustomCommandConfigMenuOptions = {
+    title: 'Cody Custom Commands (Experimental) - New User Command',
 }
 
 const chatSeparator: QuickPickItem = { kind: -1, label: 'inline chat' }
-const chatOption: QuickPickItem = { label: 'Ask a Question', alwaysShow: true }
+const chatOption: QuickPickItem = { label: 'Ask a Question' }
 const commandsSeparator: QuickPickItem = { kind: -1, label: 'commands' }
 const customCommandsSeparator: QuickPickItem = { kind: -1, label: 'custom commands' }
 const configOption: QuickPickItem = { label: 'Configure Custom Commands...' }
 const settingsSeparator: QuickPickItem = { kind: -1, label: 'settings' }
 const addOption: QuickPickItem = { label: 'New Custom Command...', alwaysShow: true }
+const submitOption: QuickPickItem = { label: 'Submit question', alwaysShow: true }
```

Comment:
Let's keep the casing consistent:

```suggestion
const submitOption: QuickPickItem = { label: 'Submit Question', alwaysShow: true }
```